src/main/groovy/com/github/prokod/gradle/crossbuild/tasks/CrossBuildPomTask.groovy

```diff
@@ -39,52 +42,82 @@ class CrossBuildPomTask extends AbstractCrossBuildPomTask {
     Tuple2<ScopeType, Configuration> createCrossBuildPomAidingConfigurationForMavenScope(ScopeType scopeType,
                                                                                          SourceSet crossBuildSourceSet){
 
-        def mavenToGradleScope = { ScopeType scope ->
-            switch (scope) {
-                case ScopeType.COMPILE:
-                    return project.configurations[crossBuildSourceSet.compileClasspathConfigurationName]
-                case ScopeType.RUNTIME:
-                    return project.configurations[crossBuildSourceSet.runtimeClasspathConfigurationName]
+        def resolveSourceSet = { ScopeType scope ->
+            def sourceSetName = {
+                switch (scope) {
+                    case ScopeType.COMPILE:
+                        return crossBuildSourceSet.compileClasspathConfigurationName
+                    case ScopeType.COMPILE_ONLY:
```

Comment:
then it was still consistent with this code.
I think there should be a scope type which tells do not include it.